umn/source/faqs/how_to_access_kibana_from_elb.rst

```diff
@@ -0,0 +1,179 @@
+.. _css_02_0201:
+
+How to access Kibana from outside cloud using ELB
```

Comment:
Is this the title? Can we add a question mark (?) here? The other FAQs have question marks so it might be good to have one here. :-)

Reply:
Added the (?) mark :)